### PR TITLE
Add wlan tuning

### PR DESCRIPTION
**Pull request by virtualguy**
* proxy_arp
* bss_transition
* fast_roaming_enabled

### CODE REVIEW

File: docs/resources/wlan.md [hunk 1]
```diff
@@ -62,6 +62,8 @@ resource "unifi_wlan" "wifi" {
 ### Optional
 
 - `ap_group_ids` (Set of String) IDs of the AP groups to use for this network.
+- `bss_transition` (Boolean) Enable BSS Transition Defaults to `false`.
```
Comment (joshuaspence) [inline docs/resources/wlan.md:65]: Add a period to the description

File: docs/resources/wlan.md [hunk 2]
```diff
@@ -62,6 +62,8 @@ resource "unifi_wlan" "wifi" {
...
+- `fast_roaming_enabled` (Boolean) Enable Fast Roaming Defaults to `false`.
```
Comment (joshuaspence) [inline docs/resources/wlan.md:66]: Also here

File: docs/resources/wlan.md [hunk 3]
```diff
@@ -75,6 +77,7 @@ resource "unifi_wlan" "wifi" {
 - `no2ghz_oui` (Boolean) Connect high performance clients to 5 GHz only Defaults to `true`.
 - `passphrase` (String, Sensitive) The passphrase for the network, this is only required if `security` is not set to `open`.
 - `pmf_mode` (String) Enable Protected Management Frames. This cannot be disabled if using WPA 3. Valid values are `required`, `optional` and `disabled`. Defaults to `disabled`.
+- `proxy_arp` (Boolean) Enable Proxy ARP Defaults to `false`.
```
Comment (joshuaspence) [inline docs/resources/wlan.md:80]: And here

File: go.mod [hunk 4]
```diff
@@ -1,4 +1,4 @@
-module github.com/paultyng/terraform-provider-unifi
+module github.com/virtualguy/terraform-provider-unifi
```
Comment (joshuaspence) [inline go.mod:1]: Unrelated?

File: internal/provider/resource_wlan.go [hunk 5]
```diff
@@ -174,12 +174,30 @@ func resourceWLAN() *schema.Resource {
...
+				Description: "Proxy ARP",
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     false,
+			},
+			"bss_transition": {
+				Description: "BSS Transition",
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     false,
```
Comment (joshuaspence) [inline internal/provider/resource_wlan.go:187]: This should default to true to match the UI

File: internal/provider/resource_wlan_test.go [hunk 6]
```diff
@@ -271,6 +271,58 @@ func TestAccWLAN_no2ghz_oui(t *testing.T) {
...
+			// TODO: actual CheckDestroy
+
+			<-wlanConcurrency
+			return nil
+		},
+		Steps: []resource.TestStep{
+			{
+				Config: testAccWLANConfig_proxy_arp(subnet, vlan),
+				Check:  resource.ComposeTestCheckFunc(
+				// testCheckNetworkExists(t, "name"),
```
Comment (joshuaspence) [inline internal/provider/resource_wlan_test.go:292]: You're not actually checking anything here

**Review by joshuaspence — CHANGES_REQUESTED** [review]

**Review by joshuaspence — APPROVED** [review]

**Comment by virtualguy** [comment]
Thanks for the clean up @joshuaspence much appreciated

**Comment by joshuaspence** [comment]
> Thanks for the clean up @joshuaspence much appreciated

No problem, I wanted to use these changes myself! Thanks for the contribution

**Pull request merged**